tests/BrightcoveOS .NET-MAPI-Wrapper.Tests/IntegrationTests/ApiKeys.cs

```diff
@@ -10,11 +10,11 @@ public static class ApiKeys
 		/// <summary>
 		/// Read token taken from Brightcove's media API sample @ http://docs.brightcove.com/en/media/samples/search_videos.html.
 		/// </summary>
-		public const string ReadToken = "ZY4Ls9Hq6LCBgleGDTaFRDLWWBC8uoXQun0xEEXtlDUHBDPZBCOzbw..";
+		public const string ReadToken = "AIe8Zmp7bAq8paSBoLt_-1_jII2yjQ1Yd1yAQxmFK10NcfVmhVhiPQ..";

```

Comment:
@kish9nine in order to test locally, you can change these values to use your own